login.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Trackbill</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/tailwindcss/2.2.19/tailwind.min.css">
+</head>
+<body>
+  
+    <!-- <nav class="bg-indigo-500 shadow-lg">
+        <div class="container mx-auto">
+          <div class="sm:flex justify-around">
+            <a href="indexs.html" class="text-white text-3xl font-bold p-3">TrackBill</a>
+            <ul class="text-grey-400 sm:self-center text-xl border-t sm:border-none">
+                <li class="sm:inline-block">
+                    <a href="indexs.html" class="p-3 hover:text-white" >Home</a>
+                </li>
+                <li class="sm:inline-block">
+                    <a href="login.html" class="p-3 hover:text-white" >Login</a>
+                </li>
+            </ul>
+          </div>
+        </div>
+      </nav>
+      <div class="min-h-screen bg-gray-50 py-6 text-gray-800 flex flex-col justify-center relative overflow-hidden sm:py-12">
+        <div class="relative py-3 sm:max-w-xl mx-auto text-center">
+      <span class="text-2xl font-bold ">
+            Login
+      </span>
+      <div class="mt-4 bg-white shadow-md rounded-lg text-left">
+          <div class="h-2 bg-indigo-400 rounded-t-md"> </div>
+          <form action="expense.html" method="post">
+              <div class="px-8 py-6">
+                <label for="" class="block font-semibold">Email:</label>  
+              <input type="email" placeholder="Enter Email" name="email" class="border-w-full h-5 px-3 py-5 md-2 hover:outline:none focus-ring-1 focus:ring-indigo-400 rounded-md">
+             
+            <div class="flex justify-center items-baseline">
+                <button type="submit" class="mt-4 bg-indigo-500 text-white py-2 px-6 rounded-md hover:bg-indigo-600">Login</button>
+            </div>
+            </div>
+        </form>
+        
```

Comment:
what's the use of these codes you commented out? Looks like the whole file is not even needed why do we have it here?